tests/resources/test_project.py

```diff
@@ -306,7 +468,84 @@ def test_project_registration(collection, session):
     assert create_time == created_project.created_at
 
 
-def test_get_project(collection, session):
+def test_failed_register_v3():
+    team_id = uuid.uuid4()
+    session = mock.Mock()
+    session.post_resource.side_effect = NotFound(f'/teams/{team_id}/projects',
+                                                 FakeRequestResponse(400))
+    project_collection = ProjectCollection(session=session, team_id=team_id)
+    with pytest.raises(ModuleRegistrationFailedException) as e:
+        project_collection.register("Project")
```

Comment:
Is the `team_id` argument supposed to be required in the v3 API?

Reply:
it's part of the collection creation

Reply:
Same request here.

Reply:
updated

Reply:
I see the update. 👍🏼